packages/devtools/pubspec.yaml

```diff
@@ -16,8 +16,8 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dependencies:
```

Comment:
you can revert these pubspec changes and just rely on pinned dependencies until we publish a new version.

Reply:
done.